src/components/CurriculumComponents/CurriculumVisualiser/CurriculumVisualiser.tsx

```diff
@@ -159,6 +159,19 @@ function isHighlightedUnit(unit: Unit, selectedThread: Thread | null) {
   return unit.threads.some((t) => t.slug === selectedThread.slug);
 }
 
+function sortChildSubjects(subjects: Subject[]) {
+  return [...subjects].sort((a, b) => {
+    // Special logic we always want combined-science first.
```

Comment:
Just a note - since we have a lot of sorts, it might be worth finding a consistent way to sort / secondary sort... Definitely something we can chat about